### PR TITLE
chore: Dev-Ex Pass

### PR DESCRIPTION
**Pull request by cnojima**
Added:

- **.node-version** to prescribe a compatible LTS for future contributors
- **.npmrc** to prevent having to use `--legacy-peer-deps` with every update
- **CONTRIBUTING.md** to start establishing project best-practices
- **GETTING_STARTED.md** to enable contributor quick-start

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+## Your first issue & pull request
+- Check current issues in the [BACKLOG](https://github.com/users/fribbels/projects/2)
+- Priority stack ranking and rough project status can be found in the [PRIORITY BOARD](https://github.com/users/fribbels/projects/2/views/2)
+- Find an issue to work on and set it status to `In Progress` by moving it from the Backlog column to the In Progress column.  *This lets other contributors know that the issue is being worked on!*
+
+After you have forked the repo and switched to the `beta` branch, create your working branch:
+```
+git checkout -b [feature|fix]/[your-branch-name]
+```
+
```
Comment (ronaldpereira) [inline CONTRIBUTING.md:43]: I would add a reference to why we create branch names like this by citing Gitflow just for completeness:
https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+## Commit Message Format
+Each commit message consists of a **header**, a **body**, and a **footer**. The header has a special format that includes a **type**, a **scope**, and a **subject**:
+
+***The header is mandatory and the scope of the header is optional.***
```
Comment (ronaldpereira) [inline CONTRIBUTING.md:47]: I usually don't like those commit patterns. The only thing that a commit message should be is concise and contain all the information to be easier to find what was done when seeing git log/commit history.
We can however recommend it, but not make it mandatory.
But as we are using github flow reference, we can cite it directly:

```txt
Commit and push your changes to your branch. Give each commit a descriptive message to help you and future contributors understand what changes the commit contains. For example, fix typo or increase rate limit.
```

https://docs.github.com/en/get-started/using-github/github-flow#make-changes

**Comment by ronaldpereira** [comment]
I'm thinking that we can also add an issue and PR templates. It's also nice to have it as a general guideline for newcomers.

**Comment by fribbels** [comment]
Good idea on the nodeversion and npmrc files! RE the commit guidelines - I'm cool with recommending it going forward, will try my best to follow it, but if it gets missed here and there then nbd. Having a good commit description is more important to me

**Pull request merged**